src/server/utilities/liteATITracking/viewTracking/index.ts

```diff
@@ -0,0 +1,25 @@
+export default async () => {
+  if (typeof window.IntersectionObserver === 'undefined') {
+    console.log('OBSERVER POLYFILL ME OUT!!!!');
+    // Polyfill IntersectionObserver, e.g. for IE11
+    await import('intersection-observer');
+  }
+
+  const options = { threshold: 0.5 };
```

Comment:
Can we add a comment to this, so that it's clear what this means? Or extract it into a const with a meaningful name?

Reply:
In fact, why don't we use a single variable but import it where required? https://github.com/bbc/simorgh/blob/bcd1df464aafb8dcd5949bb2b3227bea9fdcc11f/src/app/hooks/useViewTracker/index.jsx#L12